tests/Unit/SubPageList/Lister/SubPageListTest.php

```diff
@@ -73,6 +73,7 @@ private function getRenderedList( SubPageList $subPageList, $titleText ) {
 				'showpage' => new ProcessedParam( 'showpage', true, false ),
 				'default' => new ProcessedParam( 'default', '', true ),
 				'limit' => new ProcessedParam( 'limit', 200, true ),
+				'sort' => new ProcessedParam( 'sort', 'asc', true ),
```

Comment:
It caused to error "[Undefined index: sort](https://travis-ci.org/charger/SubPageList/jobs/213548216)" 
I think this is because of mocks.
I'm not so good at PHPUnit.